Add September 2023 newsletter (#2020)
writethedocs/www: docs/blog/newsletter-september-2023.rst

.. post:: September 05, 2023
   :tags: newsletter

##########################################
Write the Docs Newsletter – September 2023
##########################################

Hey, documentarians, the newsletter is back!

> In conference news, we have the `2023 Atlantic conference </conf/atlantic/2023/>`__ this weekend. If you don't have a ticket yet, grab one while you can (through 7 September). Looking forward to seeing you there. And after you get inspired, the Australia conference still has its `call for proposals </conf/australia/2023/cfp/>`__, but it closes 15 September, so get your ideas in soon.

Reviewer: Nit, it's starting at the weekend, but running Mon+Tue :-p

Author: Good point!
```suggestion
In conference news, we have the `2023 Atlantic conference </conf/atlantic/2023/>`__ running Sunday through Tuesday. If you don't have a ticket yet, grab one while you can (through 7 September). Looking forward to seeing you there. And after you get inspired, the Australia conference still has its `call for proposals </conf/australia/2023/cfp/>`__, but it closes 15 September, so get your ideas in soon.
```

> If you're interested in information about the Write the Docs community, take part in our `2023 Write the Docs Salary Survey <https://writethedocs.org/surveys>`__. It opens in just a few days.

Reviewer: "If you're interested in information about the Write the Docs community, " could be reworded to something stronger. 

To help the community build out stronger and fairer salary pratices... :man_shrugging:

Author: This paragraph was just a placeholder until @goatlady suggested something stronger. I wasn't sure what she wanted to emphasize.

Reviewer 2: I sent something through on Slack about a minute before I saw this :) I will add it in now...

Reviewer 2: It's almost that time again: the Documentation Salary Survey for 2023 is opening for submissions next week. We encourage everyone working in areas related to documentation to participate, whether you're working or currently unemployed, full-time or part-time, permanent or contract, emplopyee or freelance, in all countries, all industries, and no matter what level of skill or experience you possess. Your contribution will help the whole community better understand what an appropriate salary is, and provide a basis for future negotiations.

This year, we're removing the questions on workplace changes related to the pandemic, and including new questions about "return to the office" mandates, job changes and your thoughts on job security - issues at the top of a lot of people's minds. We'll be announcing when the survey goes live on Slack and Twitter, and as always you can find all the details at https://salary-survey.writethedocs.org/.

> If you're interested in information about the Write the Docs community, take part in our `2023 Write the Docs Salary Survey <https://writethedocs.org/surveys>`__. It opens in just a few days.

Reviewer: ```suggestion
It's almost that time again: the Write the Docs Salary Survey for 2023 is opening next week. We encourage everyone working in areas related to documentation to participate, whether you're working or currently unemployed, full-time or part-time, permanent or contract, employee or freelance, in all countries, all industries, and no matter what level of skill or experience you possess. Your contribution will help the whole community better understand what an appropriate salary is, and provide a basis for future negotiations.

This year, we're removing the questions on workplace changes related to the pandemic, and including new questions about "return to the office" mandates, job changes and your thoughts on job security - issues at the top of a lot of people's minds. We'll be announcing when the survey goes live on Slack and Twitter, and as always you can find all the details at https://salary-survey.writethedocs.org/.
```

Reviewer: @goatlady note the few copy edits

> I'd like to extend a special goodbye to Beth Aitman. This month marks the first newsletter in years that she hasn't participated in. Thanks for your leadership, help, and many other contributions

Reviewer 3: ```suggestion
I'd like to extend a special goodbye to Beth Aitman. This month marks the first newsletter in years that she hasn't participated in. Thanks for your leadership, help, and many other contributions.
```

> I'd like to extend a special goodbye to Beth Aitman. This month marks the first newsletter in years that she hasn't participated in. Thanks for your leadership, help, and many other contributions.

Reviewer: :-(

Reviewer 4: +1

Author: 😞

Other recommendations for further information:

- `Any friend of the docs is a friend of mine: Cultivating a community of documentation advocates </videos/portland/2019/any-friend-of-the-docs-is-a-friend-of-mine-cultivating-a-community-of-documentation-advocates-heather-stenson/>`__ from Write the Docs Portland, 2019
- `2022 State of DevOps Report data deep dive: Documentation is like sunshine <https://cloud.google.com/blog/products/devops-sre/deep-dive-into-2022-state-of-devops-report-on-documentation>`__
- `Why it’s worth it to invest in internal docs <https://increment.com/documentation/why-investing-in-internal-docs-is-worth-it/>`__

--------------------------
Numbering Section Headings
--------------------------

To number or not was a question posed in recent months. In general, these queries related to printed (or PDF) manuals.

> Often reasons for numbering sections relate to using the number instead of the section title as a reference or to clarify about sequencing (or relationships) between sections. As one documentarian noted, "Titles alone do not tell users ... where they will find the information unless they are already ... familiar with the content structure of the document. When you have to check section 3, you know that it is before section 4 and ... after section 2."

Reviewer 3: The first sentence seems a little hard to parse. Does it mean something like this?

`Often, the reasons for numbering section headings include using the number as a reference or clarifying the sequence of sections and relationships between sections.`

Author: ```suggestion
Often, the reasons for numbering section headings relate to using the number as a reference or clarifying the sequencing of sections and relationships between sections. As one documentarian noted, "Titles alone do not tell users ... where they will find the information unless they are already ... familiar with the content structure of the document. When you have to check section 3, you know that it is before section 4 and ... after section 2."
```

Several people mentioned issues about updating the content and adding in new sections. Even with automatic numbering, readers of previous versions may suggest looking at a specific section by its number – only to find that the number has changed in a recent update.

When numbering is a content convention, it may not represent a sequence to follow. In this case, numbers can interrupt the reader's focus by forcing them determine whether they really need to go through content sequentially. One person wrote, "Numbering is often one of those unnecessary things that people do without really thinking why." Documentarians may have section 3 before section 4 just because that's the sequence that they thought of – not because it needs to come first.

For online documentation, people were less in favor of using numbers. The discussion emphasized that it could depend on the structure of the online content and navigational features.

------------------------------------------------------
Dealing with Unpleasant People in Open-Source Projects
------------------------------------------------------

> Although open-source groups are famed for their collaborative attitude, dealing with problematic personalities can sometimes be challenging. A recent WtD discussion provided useful ideas for efficiently dealing with such circumstances.

Author: ```suggestion
Although open-source groups are famed for their collaborative attitude, dealing with problematic personalities can sometimes be challenging. A recent WTD discussion provided useful ideas for efficiently dealing with such circumstances.
```

> In this conversation, advice on how to deal with an individual whose behavior in an open-source project was troublesome. This person acted in ways that on the verge of being unacceptable without being overtly prohibited. Their perceived importance in the community added to the complication.

Reviewer 3: I think the first sentence is incomplete - there seems to be something missing about the advice in the conversation. Or does it mean that the advice was troublesome?

Author: Good point. 🤦 
```suggestion
In this conversation, advice was needed on how to deal with an individual whose behavior in an open-source project was troublesome. This person acted in ways that verged on being unacceptable without being overtly prohibited. Their perceived importance in the community added to the complication.
```

> In this conversation, advice on how to deal with an individual whose behavior in an open-source project was troublesome. This person acted in ways that on the verge of being unacceptable without being overtly prohibited. Their perceived importance in the community added to the complication.

Reviewer 3: ```suggestion
In this conversation, advice on how to deal with an individual whose behavior in an open-source project was troublesome. This person acted in ways that verged on being unacceptable without being overtly prohibited. Their perceived importance in the community added to the complication.
```

Moreover, a more empathic strategy was suggested, such as advocating for compassion rather than aggressiveness in response. Recognizing the content of their comments while politely expressing disagreement can pave the way for meaningful discourse.

Finally, it was emphasized that rules of conduct promote a culture of respect and kindness and that leading with the assumption that the person didn’t intend to be impolite can help ease tensions.

Fostering a communal feeling of respect and collaboration is critical in open-source groups. By following standards of behavior and encouraging polite debate, these communities may continue to grow, guaranteeing an inclusive and productive environment for all contributors. Though difficult, dealing with challenging people is critical to sustaining the integrity and vibrancy of open-source initiatives.

-----------------------------------------
Keeping Up-To-Date with Necessary Changes
-----------------------------------------

> Even in periods when many people are taking vactions, an organization can have a lot going on. A recent discussion touched on how to stay on top of it all when you have many articles to keep track of. Some complained about working long hours to add documentation only to be told right after it was published that it no longer applied. The discussion turned to tools and processes to avoid such issues.

Author: ```suggestion
Even in periods when many people are taking vacations, an organization can have a lot going on. A recent discussion touched on how to stay on top of it all when you have many articles to keep track of. Some complained about working long hours to add documentation only to be told right after it was published that it no longer applied. The discussion turned to tools and processes to avoid such issues.
```

Many people used a ticketing system. Jira was popular because it was where others were working and so made it easier to follow changes. For similar reasons, others used GitHub or GitLab and followed changes directly in projects. Others had their own separate system in someplace like Asana with just what they needed.

> Outside of tools, people suggested setting up regular meetings with people like product managers, especially if these meetings line up with a release schedule. It helped to put in some sort of check of whether doc changes are necessary before release. Others suggested seeing if you can get rid of some content, such as pages with few visits. Then there's less to deal with.

Author: ```suggestion
Other than tools, people also suggested setting up regular meetings with people like product managers, especially if these meetings line up with a release schedule. It helped to put in place some sort of check to determine whether doc changes are necessary before release. Others suggested seeing if you can get rid of some content, such as pages with few visits. Then there's less to deal with.
```

Even with the best process, everyone misses some things. So it helps to think about how to find the things you've missed. For example, make it easy to report issues so you learn about anything that falls through. And multiple people noted the benefits of regular documentation audits, starting with the most important or most visited pages. Go through things slowly but steadily to check that everything still works as it should.

----------------

> From our sponsor

Author: ```suggestion
From Our Sponsor
```

ClickHelp is a comprehensive cloud-based documentation platform for efficient teamwork. Its robust capabilities empower users to efficiently generate, review, and release documentation. The platform optimizes workflows through features like single sourcing and dynamic output, enhancing the overall process. Furthermore, it is flexible and customizable, with a fast and accurate import process. The tool promotes seamless collaboration and effortless version tracking. It is accessible across various devices and platforms, accommodating multiple formats.

Noteworthy functionalities include OpenAPI (Swagger) integration, seamless migration from Confluence, powerful navigation elements, effective translation management, and many more. Embark on your journey of exploration by initiating a free trial or scheduling a demo to experience its cutting-edge features. Join us on this transformative expedition with ClickHelp.

*Interested in sponsoring the newsletter? Take a look at our* `sponsorship prospectus </sponsorship/newsletter/>`__.

----------------

> Events Coming Up

Reviewer: It would be a manual step, but we could include CFP ends, and conferences in this list.

.. post:: September 05, 2023
   :tags: newsletter

##########################################
Write the Docs Newsletter – September 2023
##########################################

Hey, documentarians, the newsletter is back!

In conference news, we have the `2023 Atlantic conference </conf/atlantic/2023/>`__ running Sunday through Tuesday. If you don't have a ticket yet, grab one while you can (through 7 September). Looking forward to seeing you there. And after you get inspired, the Australia conference still has its `call for proposals </conf/australia/2023/cfp/>`__, but it closes 15 September, so get your ideas in soon.

> It's almost that time again: the Write the Docs Salary Survey for 2023 is opening next week. We encourage everyone working in areas related to documentation to participate, whether you're working or currently unemployed, full-time or part-time, permanent or contract, employee or freelance, in all countries, all industries, and no matter what level of skill or experience you possess. Your contribution will help the whole community better understand what an appropriate salary is, and provide a basis for future negotiations.
> 
> This year, we're removing the questions on workplace changes related to the pandemic, and including new questions about "return to the office" mandates, job changes and your thoughts on job security - issues at the top of a lot of people's minds. We'll be announcing when the survey goes live on Slack and Twitter, and as always you can find all the details at https://salary-survey.writethedocs.org/.

Author: I'd prefer a single paragraph, since the newsletter is already rather long as is. So it'd be a little shorter this way.

```suggestion
Elsewhere, the Write the Docs Salary Survey for 2023 is opening next week. We encourage everyone working in areas related to documentation to participate. Your contribution will help the whole community better understand what an appropriate salary is and provide a basis for future negotiations. This year, we're removing the questions on workplace changes related to the pandemic and including new questions about "return to the office" mandates, job changes, and your thoughts on job security – issues at the top of a lot of people's minds. We'll be announcing when the survey goes live on Slack and Twitter, and you can always find all details at https://salary-survey.writethedocs.org/.
```

Reviewer 2: I'd like to keep the mention of unemployed people and freelance/contract in - how about this?

Elsewhere, the Write the Docs Salary Survey for 2023 is opening next week. We encourage everyone working in areas related to documentation to participate - working or currently unemployed, contract or permanent, employee or freelance. Your contribution will help the whole community better understand what an appropriate salary is and provide a basis for future negotiations. This year, we're including new questions about "return to the office" mandates, job changes, and your thoughts on job security – issues at the top of a lot of people's minds. We'll be announcing when the survey goes live on Slack and Twitter, and you can always find all details at https://salary-survey.writethedocs.org/.

Author: ```suggestion
Elsewhere, the Write the Docs Salary Survey for 2023 is opening next week. We encourage everyone working in areas related to documentation to participate - working or currently unemployed, contract or permanent, employee or freelance. Your contribution will help the whole community better understand what an appropriate salary is and provide a basis for future negotiations. This year, we're including new questions about "return to the office" mandates, job changes, and your thoughts on job security – issues at the top of a lot of people's minds. We'll be announcing when the survey goes live on Slack and Twitter, and you can always find all details at https://salary-survey.writethedocs.org/.
```
